scout/server/blueprints/cases/controllers.py

```diff
@@ -794,6 +795,39 @@ def rerun(store, mail, current_user, institute_id, case_name, sender, recipient)
         LOG.error("Cannot send rerun message: no recipient defined in config.")
 
 
+def call_rerunner(store, institute_id, case_name, metadata):
+    """Call rerunner with updated pedigree metadata."""
+    LOG.info("Building query for reanalysis")
+
+    # define the data to be passed
+    payload = {"case_id": case_name, "sample_ids": [m["sample_id"] for m in metadata]}
+
+    cnf = rerunner.connection_settings
+    url = f"http://{cnf.get('host')}/v1.0/rerun"
```

Comment:
This line makes the service really customized for your facility. When we are going to have our rerunner service in place we will be forced to have its url ending in /v1.0/rerun otherwise it won't work. I think url should be what you pass in the config file and set in the rerunner extensions.

Reply:
The simplest way to fix this would be to pass in the config file the complete url: "http://someurl:port/v1.0/rerun", so you won't need to fix the port in the rerunner extensions settings and it would work directly